TheMET/TheMET/ViewControllers/CultureArtsViewController.swift

```diff
@@ -72,13 +53,34 @@ class CultureArtsViewController: UIViewController, UITableViewDelegate, UITableV
         self.culturesTableView.delegate = self
         self.searchBar.delegate = self
         self.failedCategoryView.onButtonTap = { [weak self] in
-            self?.reloadButtonDidTap()
+            self?.viewModel?.reloadButtonDidTap()
+            self?.culturesTableView.reloadData()
         }
         self.updateContent()
-        self.reloadCategory()
+        self.viewModel?.reloadCategory()
```

Comment:
`reloadCategory` method must be private and can't be used executed from `ViewController cause this is not a UI event or user action.